LG-Gram_17-20231210-201210.yml

```diff
@@ -0,0 +1,52 @@
+---
+layout: ‘hcl’
```

Comment:
You need to replace all `‘` and `’` with `'`. This happens when you copy and paste from the forum and the OP didn't put the report into a code block. My recommendation is to edit the forum post first by adding ` ``` ` before and after the HCL report. That puts it into a code block and copy & paste will function correctly then.